lib/hoodaw_utils.rb

```diff
@@ -13,6 +14,30 @@ def commify(value)
     with_commas = whole.gsub(/\B(?=(...)*\b)/, ",")
     [with_commas, decimal].join(".")
   end
+
+  def link_to_github_team(team_name)
+    if !team_name.nil? 
+      link_to(team_name.to_s,GITHUB_TEAM_URL+team_name.to_s)
+    end
+  end
+
+  def links_to_domain_names(domain_names)
+    domain_names.map { |domain_name| link_to(domain_name,"https://"+domain_name) }
+                .join("<br/> ")
+  end
+
+  def link_to_repo(text, href)
+    "<a href='#{href}'>#{split(text)}</a>" 
```

Comment:
Because "split" is a built-in method on string, I think it's a bit confusing to have a separate "split" method which specifically splits on "/" So, I'd probably remove that method, since it's only used here, and rewrite this as:

```
def link_to_repo(repo, url)
  name = repo.split("/").last
  link_to(name, url)
end
```

IMO, this keeps the intention clear, and it means the *only* place you're actually building `<a href=...` tags is inside the `link_to` helper, which feels like a good thing.

Reply:
Fair point @digitalronin . I have updated the code.